cryptomesh/scripts/policyindexer.py

```diff
@@ -0,0 +1,23 @@
+import asyncio
+from cryptomesh.orchestrator.orchestrator import orchestrate
+from cryptomesh_client.client import CryptoMeshClient
+from cryptomesh.log.logger import get_logger
+
+logger = get_logger(__name__)
+
+async def main():
+    try:
+        client = CryptoMeshClient(base_url="http://localhost:19000", token="my-secret")
+        await orchestrate("policies/example.yml", client)
```

Comment:
orchestrate debe ser un metodo de la clase CryptoMeshClient, al parecer ya se habia sugerido el uso del cliente en el issue

from cryptomesh_client import CryptoMeshClient

client = CryptoMeshClient(base_url="http://localhost:8000", api_key="my-secret")
client.interpret("full_policy.yml")

Aqui interpret es un metodo de CryptoMeshClient y se le pasa como parametro el archivo .yml


Te dejo un ejemplo de como puedes crear este Metodo en tu cliente 
class CryptoMeshClient:
    def __init__(self, base_url: str, api_key: str):
        self.base_url = base_url
        self.api_key = api_key
        self.client = httpx.AsyncClient(headers={"Authorization": f"Bearer {api_key}"})

    async def interpret(self, policy_path: str):
        # Interpretar el archivo YAML
        with open(policy_path, "r") as f:
            data = yaml.safe_load(f)
        
        # Lógica para orquestar (ejemplo de POST al servidor)
        
```suggestion
        await orchestrate("policies/example.yml", client)
```

Tambien sugiero renombrar tu Metodo de orchestrate a load_yaml, interpret, etc.